classes/Kohana/Jam/Price.php

```diff
@@ -250,19 +252,19 @@ public function is($operator, $value)
 		switch ($operator) 
 		{
 			case Jam_Price::EQUAL_TO:
-				$result = ($this->amount() == $value);
+				$result = (abs($this->amount() - $value) < Jam_Price::EPSILON);
 			break;
 			case Jam_Price::GREATER_THAN:
-				$result = ($this->amount() > $value);
+				$result = (($this->amount() - $value) >= Jam_Price::EPSILON);
 			break;
 			case Jam_Price::LESS_THAN:
-				$result = ($this->amount() < $value);
+				$result = (($value - $this->amount()) >= Jam_Price::EPSILON);
 			break;
 			case Jam_Price::GREATER_THAN_OR_EQUAL_TO:
-				$result = ($this->amount() >= $value);
+				$result = (($value - $this->amount()) < Jam_Price::EPSILON);
 			break;
 			case Jam_Price::LESS_THAN_OR_EQUAL_TO:
-				$result = ($this->amount() <= $value);
+				$result = (($this->amount() - $value) < Jam_Price::EPSILON);
```

Comment:
You're right I didn't notice the argument order change. It's not that noticeable :stuck_out_tongue: